Cargo.toml

```diff
@@ -28,3 +28,5 @@ categories = ["web-programming"]
 [dependencies]
 bytes = "1"
 http = "0.2"
+pin-project-lite = "0.2"
+futures-core = "0.3"
```

Comment:
In thought of this I know we will get `Stream` soon so lets make an issue around tracking a 1.0 release of `http-body`. What do you think @seanmonstar?

Reply:
I've removed `IntoStream` for now.